guides/source/getting_started.md

```diff
@@ -7,1150 +7,1558 @@ This guide covers getting up and running with Ruby on Rails.
+
+```css
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+nav {
+  display: flex;
+  gap: 0.5rem;
+}
+
+img {
+  max-width: 800px;
+}
+
+.notice {
+  color: green;
+}
+```
+
+Refresh your page and you'll see the CSS has been applied.
+
+### Import Maps
+
+Rails uses import maps for JavaScript by default. This allows you to write modern JavaScript modules with no build steps.
+
+You can find the JavaScript pins in `config/importmap.rb`. This file maps the JavaScript package names with the source file which is used to generate the importmap tag in the browser.
+
+```ruby
+# Pin npm packages by running ./bin/importmap
+
+pin "application"
+pin "@hotwired/turbo-rails", to: "turbo.min.js"
+pin "@hotwired/stimulus", to: "stimulus.min.js"
+pin "@hotwired/stimulus-loading", to: "stimulus-loading.js"
+pin_all_from "app/javascript/controllers", under: "controllers"
+```
+
+TIP: Each pin maps a JavaScript package name (e.g., `"@hotwired/turbo-rails"`) to a specific file or URL (e.g., `"turbo.min.js"`). `pin_all_from` maps all files in a directory (e.g., `app/javascript/controllers`) to a namespace (e.g., `"controllers"`).
+
+Import maps keep the setup clean and minimal, while still supporting modern JavaScript features.
+
+What are these JavaScript files already in our import map? They are a frontend framework called Hotwire that Rails uses by default.
+
+### Hotwire
+
+Hotwire is a JavaScript framework designed to take full advantage of server-side generated HTML. It is comprised of 3 core components:
+
+1. [**Turbo**](https://turbo.hotwired.dev/) handles navigation, form submissions, page components, and updates without writing any custom JavaScript.
+2. [**Stimulus**](https://stimulus.hotwired.dev/) provides a framework for when you need custom JavaScript to add functionality to the page.
+3. [**Native**](https://native.hotwired.dev/) allows you to make hybrid mobile apps by embedding your web app and progressively enhancing it with native mobile features.
+
+We haven't written any JavaScript yet, but we have been using Hotwire on the frontend. For instance, the form you created to add and edit a product was powered by Turbo.
+
+Learn more in the [Asset Pipeline](asset_pipeline.html) and [Working with JavaScript in Rails](working_with_javascript_in_rails.html) guides.
+
+Rich Text Fields with Action Text
+---------------------------------
+
+Many applications need rich text with embeds (i.e. multimedia elements) and Rails provides this functionality out of the box with Action Text.
+
+To use Action Text, you'll first run the installer:
+
+```bash
+$ bin/rails action_text:install
+$ bundle install
+$ bin/rails db:migrate
+```
+
+Restart your Rails server to make sure all the new features are loaded.
+
+Now, let's add a rich text description field to our product.
+
+First, add the following to the `Product` model:
+
+```ruby#2
+class Product
+  has_rich_text :description
```

Comment:
Where did we create the description column? Would this work without the column?

Reply:
This is using ActionText, so it doesn't need it's own column since it uses the ActionText tables.